dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -1869,10 +1869,15 @@ namespace {
 
   bool
   is_bit(const char* topic_name) {
+#if !defined (DDS_HAS_MINIMUM_BIT)
```

Comment:
Could this be refactored to use the existing function topicIsBIT in dds/DCPS/BuiltInTopicUtils.h?